nlptest/transform/robustness.py

```diff
@@ -692,3 +691,61 @@ def search_contraction(text):
                     sample.transformations = transformations
             sample.category = "robustness"
         return sample_list
+        
+class NumberToWord(BaseRobustness):
+    alias_name = "number_to_word"
+    infEng = engine()
+
+    @staticmethod
+    def transform(sample_list: List[Sample]) -> List[Sample]:
+        """
+        Transform a list of strings to their equivalent verbal representation
+        of numbers present in the string.
+        Args:
+            sample_list: List of sentences to apply robustness.
+        Returns:
+            List of sentences that have numbers in their verbal representation.
+        """
+        for sample in sample_list:
+            results = []
+            trans = []
+            transformations = []
+            start_offset = 0
+            for token in sample.original.split():
+                if re.match(r'^\d+$', token):
```

Comment:
Inflect also supports decimals, should we also utilize that? It may have some side effects like changing a clock like 5.30 but I think 5:30 is more common and correct. Can you think of any other unwanted replacements this could cause? @luca-martial

Reply:
@alytarik @Ryzxxl yes we should utilize that from the inflect library. The unwanted replacements are ok, it is part of the things we have to accept

Reply:
Please have a look at some sample test cases, 

![image](https://user-images.githubusercontent.com/71117423/236560149-bc57c58d-0b8a-49c5-a6ac-6d8eb753310a.png)